<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataGenerator</class>
 <widget class="QDialog" name="DataGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Standard Dataset Generator</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Data</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="countSpin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string># of Samples</string>
   </property>
  </widget>
  <widget class="QLabel" name="dimLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dim</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="dimSpin">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="generatorCombo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>120</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Checkerboard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Concentric Circles</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spirals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sinc</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cosine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Swiss Roll</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Generator Type</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="gridCountSpin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="gridCountLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Grid Count</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="radiusSpin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="radiusLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="classesCount">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="classesLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Classes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
